src/utils/fill-grid/index.ts

```diff
@@ -10,38 +10,39 @@ import {
 
 const numbers: NUMBERS[] = [1, 2, 3, 4, 5, 6, 7, 8, 9]
 
-/**
- * A backtracking/recursive function to check all the possible combinations of numbers a solution is found
- * @param grid  9X9 Sudoku Grid
- */

```

Comment:
Any reason you removed your documentation?

Reply:
not see it necessary after  look to all file and function

Reply:
maybe I'm not right